README.md

```diff
@@ -113,8 +113,6 @@ to see all available versions.
 | Name             | Mail Address                | GitHub Profile                                |
 |------------------|-----------------------------|-----------------------------------------------|
 | Nicholas Strydom | nstrydom@gmail.com          | [nstrydom2](https://github.com/nstrydom2)     |
-| hentai-chan      | dev.hentai-chan@outlook.com | [hentai-chan](https://github.com/hentai-chan) |
-| Stefan Greve     | greve.stefan@outlook.jp     | [StefanGreve](https://github.com/stefangreve) |
```

Comment:
If this collaborator has very minimal involvement, then removal should be fine. Github tracks all contributions, so their work will still be transparent.